modular_robot/revolve2/modular_robot/body/_module.py

```diff
@@ -251,6 +251,14 @@ def color(self) -> Color:
         """
         return self._color
 
+    def set_color(self, color: Color) -> None:
```

Comment:
Shouldn't we use @color.setter ? As color is a @property ?

Reply:
indeed good catch!